README.md

```diff
@@ -46,310 +46,12 @@ prior to 2.7.1.
 
 All options and configuration can be done through interacting with the parameters
 on the main rabbitmq class.
-These are now documented via [Puppet Strings](https://github.com/puppetlabs/puppet-strings)
```

Comment:
This could still be a good hint.

Reply:
agreed. I was on the fence about this; put it back in.